radix_sort.py

```diff
@@ -0,0 +1,60 @@
+def radsort(unslist):
+    """Returns a sorted list. Accepts only a list containing positive
+    integers."""
+    # find max for iterative solution
+    maxval = max(unslist)
+    ntimes = len(str(maxval))
+
+    slist = unslist[:]
+
+    for n in range(ntimes):
+        # Making radix bins
+        bins = [[] for _ in range(10)]
+
+        # Place each list item in appropriate bin
+        for i, item in enumerate(slist):
+            inspecting = slist[i]
+            digval = _get_nth_digit(inspecting, n)
+            bins[digval].append(inspecting)
+
+        slist = []
+        # Flatten bins to list
+        for bin in bins:
+            slist.extend(bin)
```

Comment:
this works, for a fun, Pythonic way to do it, check out the `itertools` library:

```python
import itertools
return [i for i in itertools.chain(*slist)
```